### ISSUE TITLE
AdGuardHome binary ignores its working direcotry

### ISSUE THREAD

**Issue by flipreverse** [AdguardTeam/AdGuardHome#6934]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, ARM64

### Installation

GitHub releases or script from README

### Setup

On a router, DHCP is handled by the router

### AdGuard Home version

v0.107.48

### Action

Put my binary in `/usr/local/bin/adguardhome`.
I installed AdGuardHome using `/usr/local/bin/adguardhome -s install` in `/etc/adguardhome`. The systemd unit was installed as expected.

```
[Unit]
Description=AdGuard Home: Network-level blocker
ConditionFileIsExecutable=/usr/local/bin/adguardhome

After=syslog.target network-online.target 

[Service]
StartLimitInterval=5
StartLimitBurst=10
ExecStartPre=/bin/mkdir -p /var/log/
ExecStart=/usr/local/bin/adguardhome "-s" "run"

WorkingDirectory=/etc/adguardhome



StandardOutput=file:/var/log/AdGuardHome.out
StandardError=file:/var/log/AdGuardHome.err

Restart=always

RestartSec=10
EnvironmentFile=-/etc/sysconfig/AdGuardHome

[Install]
WantedBy=multi-user.target
```

### Expected result

I expect AdGuardHome to place its `data` directory  in `/etc/adguardhome` since its working directory points to that directory.

### Actual result

Although the working directory is explicitly set in that unit, it does not use it. The `data` directory is placed in `/usr/local/bin`. I can fix this by adding ` -w "."`  to the `ExecStart` directive.

**Comment by ElTopo** [comment]
I think you are confused with systemd service's `WorkingDirectory `and AdGuardHome's work-dir.

systemd service's WorkingDirectory: the directory that systemd uses to launch service
https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#WorkingDirectory=

AdGuardHome's work-dir: the directory that AdGuardHome uses to store data. From the code, either you use -w option, or the executable file path
https://github.com/AdguardTeam/AdGuardHome/blob/48c6242a7bd90ff5cf48f66a8a14d3e1e08666cb/internal/home/home.go#L780

Basically, they are different things... unless the developer decides to use systemd service's `WorkingDirectory ` as AdGuardHome's work-dir.

**Comment by flipreverse** [comment]
> Hi @flipreverse, thanks for the report. Can you confirm which OS you're using?

I see this behavior on Debian Testing arm64 (aka Trixie) as well as on Debian 12 arm64 (bookworm aka Raspbian) .
